src/__tests__/VariableSizeGrid.js

```diff
@@ -206,6 +206,19 @@ describe('VariableSizeGrid', () => {
   });
 
   describe('resetAfterIndex method', () => {
+    it('should not call forceUpdate if shouldForceUpdate is false', () => {
+      const rendered = ReactTestRenderer.create(
+        <VariableSizeGrid {...defaultProps} />
+      );
+      rendered.getInstance().forceUpdate = jest.fn();
+      rendered.getInstance().resetAfterIndices({
+        columnIndex: 5,
+        rowIndex: 15,
+        shouldForceUpdate: false,
+      });
+      expect(rendered.getInstance().forceUpdate).toHaveBeenCalledTimes(0);
```

Comment:
I think the way the "_should recalculate the estimated total size_" test checks for this is a bit more robust. (It checks to see if the `columnWidth` or `rowHeight` getter functions have been called.)

`forceUpdate` is kind of an implementation detail (e.g. maybe the grid re-renders using `setState` instead?)